sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.3.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.3.0 (2024-06-07)
 
 ### Other Changes
 
+* Includes all changes from 1.3.0-beta.1 and 1.3.0-beta.2
```

Comment:
This is the recommended message for a stable release with no additional changes.
https://azure.github.io/azure-sdk/policies_releases.html#changelog-entries-for-stable-releases